src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -222,6 +225,11 @@ public SharedTableBuilder withGetRecordsTimeLimit(long getRecordsTimeLimit) {
         return this;
     }
 
+    public SharedTableBuilder withBackupSupport(String region, String backupS3BucketName) {
```

Comment:
this is the hook for a client would configure backup support. Note, this implementation is very much tied to our shared table implementation, but I see no reason why we can't get this to work against the other two as well (byTable and byAccount), just more work to prioritize into the backlog.